docs/one-click-integrations/github-integration.md

```diff
@@ -14,32 +14,35 @@ Pulse integrates with GitHub Cloud to receive data about changes and deployments
 
 ## Setting up the GitHub integration
 
+!!! note
+    -   You can only install the Pulse GitHub App on an **organization** and not on your personal account.
+    -   [See below](#gh-permissions) the permissions that Pulse requires from your GitHub account.
+
 To set up the GitHub integration:
 
 1.  On Pulse, [expand **Integrations** and select **GitHub**](https://app.pulse.codacy.com/integrations/github){: target="_blank"}.
 
-1.  Click **Install GitHub App** and follow the instructions on the GitHub UI to install the app on your organization.
-
-    !!! important
-        You can only install the Pulse GitHub App on an organization and not on your personal account.
+1.  Click **Install GitHub App** and follow the instructions on the GitHub UI to install the app on your **organization**.
 
     ![Installing the Pulse GitHub App](images/github-installing.png)
 
-1.  Wait until you get a confirmation that Pulse successfully created the integration and the webhook on GitHub.
+1.  Choose the strategy to detect deployments that best fits your workflows. See the [section below](#deployment-detection-strategy) for a detailed description of each option.<!--TODO: Update image-->
 
-    ![Pulse GitHub integration set up successfully](images/github-ok.png)
+    ![Choosing a deployment detection strategy](images/github-strategy.png)
 
-    If there is an error please [contact support](mailto:pulsesupport@codacy.com).
+1.  Click **Complete setup**.
 
-1.  Choose the strategy to detect deployments that best fits your workflows, or turn off the automatic deployment detection. See the section below for a detailed description of each option.
+1.  Wait until you get a confirmation that Pulse successfully created the integration and the webhook on GitHub.<!--TODO: Confirm message and update image-->
 
-    ![Choosing a deployment detection strategy](images/github-strategy.png)
+    ![Pulse GitHub integration set up successfully](images/github-ok.png)
+
+    If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 
 ## Automatic deployment detection strategies {: id="deployment-detection-strategy"}
 
 The following is a detailed description of how the Pulse GitHub integration automatically detects deployment using each detection strategy:
 
-### Use merged pull requests (default)
+### Use merged pull requests (based on default branch)
```

Comment:
I think the default meant that this is the default strategy 😁

Reply:
Hummm... I used the name of the option on the UI, according to the [mockup](https://www.figma.com/file/qWJnZBS0q3YRwrQXMyRwGh/%5BPUL-623%5D-Semantic-Versioning-historical-import?node-id=1%3A171&t=MYpjL5fwdnAXLQdX-0). 
I can remove the "default" part here, if it's confusing. WDYT?

Reply:
@mrfyda After all, in the development environment we have "(default)" (which is different from the mockup). I left a comment for @nadiia-balaian [in Figma](https://www.figma.com/file/qWJnZBS0q3YRwrQXMyRwGh?node-id=1842:10086#366964241). 🙏